test/lib/server.test.js

```diff
@@ -185,7 +198,18 @@ describe('server case', () => {
         });
 
         it('doesnt affect non-errors', () => (
-            hapiEngine({ ecosystem }).then(server => (
+            hapiEngine({
+                ecosystem,
```

Comment:
I think this parameter object should be `const` or `let` and should be reusable.

Reply:
OK, I'll fix it.